src/web/Clipboard.ts

```diff
@@ -15,7 +16,7 @@ export class Clipboard extends RX.Clipboard {
         let node = Clipboard._createInvisibleNode();
         // Replace carriage return /r with /r/n, so that pasting outside browser environment
         // (eg in a native app) preserves this new line
-        text = text.replace(/\r/g, '\r\n');
+        text = _.escape(text).replace(/\r/g, '<br/>');
```

Comment:
That regexp is not sufficient for all platforms. Perhaps it should be something like `/\n\r|\r\n|\n|\r/g`. Also the comment above it is obsolete after this change.

Reply:
Thanks for your comment, Marat. 
I have updated the regexp so it works for all three cases (cr, lf, and crlf). Haven't heard of lfcr, so I did not added this case.

Reply:
Well, theoretically: https://en.wikipedia.org/wiki/Newline. Anyway yours should work for our platforms.